js-core-1/week-3/lesson.md

```diff
@@ -15,11 +16,49 @@
 By the end of this class, you should be able to answer these questions:
 
 * What is a callback and how do you use it?
+* What are anonymous functions?
 * How do you use array methods and properties to manipulate arrays?
 * How can you chain array methods?
 
 ---
 
+## Callbacks
```

Comment:
@sansaid I think this is a great idea.

We could start with explaining functions as values, as this is integral - and a way to use functions as values is callbacks.

What do you think?